cycledash/static/js/examine/utils.js

```diff
@@ -36,16 +36,17 @@ function makeIGVLink(run, igvHttpfsUrl) {
   }
 
   var nameFilePairs = [
```

Comment:
This name doesn't match the value anymore; it could be clearer to have an array called 'tracks' of objects with keys like `name`, `type`, and `uri`.